packages/office-ui-fabric-react/src/MarqueeSelection.ts

```diff
@@ -1,3 +1,2 @@
 export * from './components/MarqueeSelection/MarqueeSelection';
 export * from './components/MarqueeSelection/MarqueeSelection.types';
-export * from './utilities/selection/index';
```

Comment:
Technically a breaking change.. I would tread very carefully. Get sign off from David and Micah and make sure we maximize communication + be able to justify the change.

Reply:
Tagging @dzearing  and @micahgodbolt